nbd/src/main.cpp

```diff
@@ -88,6 +88,7 @@ static void Usage() {
         << "  --timeout <seconds>     Set nbd request timeout\n"
         << "  --try-netlink           Use the nbd netlink interface\n"
         << "Unmap options:\n"
+        << "  -f, --force                 Force unmap even if the device is mounted\n"              // NOLINT
```

Comment:
force unmap will lead to filesystem readonly

Reply:
> force unmap will lead to filesystem readonly

yeap, it will lead to filesystem become readonly.

BTW: I have constructed a test, that client write one 4K IO not success (RPC error) and keep RPC retrying, and then i do the `umount` and `curve-nbd unmap` operation, the result is:

* we can't umount the device, even if we specify `-f` option
* but we can do `curve-nbd unmap` success
* after we do the `curve-nbd unmap` operation, the filesystem is still on `read-write` mode, but if we read/write on the filesystem, it will become `read-only` mode.